File: santefluxpatientduval-duval_version/OptifluxApp/src/app/models/box_plan.ts
export interface BoxPlan {
    id?: string;
    staff_id: string;
    doctors_id: string[];
    poll: string;
    room: string;
    period: string;
    date: string;
    consultation_number: string;
    consultation_time: string;
    comment: string;
    status: string;
    created_at?: string;
    updated_at?: string;
    matricule?: string;
  }

File: santefluxpatientduval-duval_version/OptifluxAPI/schemas/boxPlan.py
from pydantic import BaseModel
from datetime import date, time, datetime
from typing import List

class BoxPlanCreate(BaseModel):
    staff_id: str
    doctors_id: List[str]
    poll: str
    room: str 
    period: str
    date: str   
    comment: str
    consultation_number: str
    consultation_time: str = None
    created_at: datetime = None
    updated_at: datetime = None
    matricule: str = None
    status: str = "Réservé"
